ci-operator/step-registry/ipi/install/install/ipi-install-install-ref.yaml

```diff
@@ -11,5 +11,10 @@ ref:
     env: OPENSHIFT_INSTALL_RELEASE_IMAGE_OVERRIDE
   - name: "release:latest"
     env: RELEASE_IMAGE_LATEST
+  env:
+  - name: "CLUSTER_VARIANT"
+    default: "default"
```

Comment:
Is there a cluster variant called "default"? I don't think so
Maybe empty string is a better default value

If you want to disconnect this from the other ``CLUSTER_VARIANT`` defined in the template, then you might be better using boolean fields (e.g. ``SINGLE_NODE=true|false``)

Reply:
Yeah I guess that would be less confusing. I'll do that